conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -26,9 +26,12 @@ show_channel_urls: true
 CONDARC
 )
 
+HOST_USER_ID=$( hash docker-machine 2>/dev/null && docker-machine ssh $(docker-machine active) 'id -u $(whoami)' || id -u $(whoami) )
```

Comment:
Sorry for being a bit slow on this, but how come just `id -u $(whoami)` doesn't cut it? What does the docker-machine part add here? (in summary: comment needed 😄 )